vtex/faststore
# Create CMS Mapping for Sections

## What's the purpose of this pull request?

Add CMS mapping for all default sections in our starter.

## How to test it?

Install this version of Faststore on your [starter](https://github.com/vtex-sites/starter.store), run `vtex cms sync` and see if everything is in place or use the preview link.

[Link Preview](https://arthurspace2--storeframework.myvtex.com/admin/new-cms/faststore)
